.chloggen/ottl-substring-intgetter.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
should this be `breaking` instead?

Reply:
good question, I think it can be `breaking`, given the start/length argument validation is done during execution phase instead of build phase.

Reply:
breaking as in API change breaking as you're changing 2 exported fields of an exported struct.